site/src/pages/components/enhanced-range-input.explainer.mdx

```diff
@@ -0,0 +1,475 @@
+---
```

Comment:
We have a slider research page already at some point it might be worth deciding what to do with that (potentially link to it from this explainer?)

Reply:
I will add it to the questions. The idea is to turn those into issues.